Lib/fontTools/misc/plistlib.py

```diff
@@ -378,42 +378,39 @@ def _string_or_data_element(raw_bytes: bytes, ctx: SimpleNamespace) -> etree.Ele
         return _string_element(string, ctx)
 
 
-@singledispatch
-def _make_element(value: Any, ctx: SimpleNamespace) -> NoReturn:
-    raise TypeError("unsupported type: %s" % type(value))
-
-
+# The following is to pacify type checkers. At the time of this writing, neither
+# mypy nor Pyright can deal with singledispatch properly. Mypy additionally
+# complains about a single overload when there should be more (?) so silence it.
 PlistEncodable = Union[
-    str,
     bool,
-    Integral,
+    bytes,
+    Data,
+    datetime,
     float,
-    Mapping[str, Any],
+    Integral,
     List[Any],
+    Mapping[str, Any],
```

Comment:
instead of `Any` here and below, should you not use (recursively) `"PlistEncodable"`?